relay/nakama/main.go

```diff
@@ -44,7 +44,7 @@ const (
 	EnvCardinalAddr      = "CARDINAL_ADDR"
 	EnvCardinalNamespace = "CARDINAL_NAMESPACE"
 
-	cardinalCollection = "cardinal_collection"
+	cardinalCollection = "cardinalCollection"
```

Comment:
NOTE: While most of this PR is just documentation changes, this is a breaking change for any Nakama users.